holidays/groups/islamic.py

```
@@ -222,6 +222,16 @@ def _add_fatima_death_day(self, name) -> set[date]:
             name, self._islamic_calendar.fatima_death_dates(self._year)
         )
 
+    def _add_grand_magal_of_touba(self, name) -> set[date]:
+        """
+        Annual relgious pilgrimage of Sengalese Mouride brotherhood.
+
+        https://en.wikipedia.org/wiki/Grand_Magal_of_Touba
+        """
+        return self._add_islamic_calendar_holiday(
+            name, self._islamic_calendar.grand_magal_of_touba_dates(self._year)
```

Pylint (3.3.7)

[error] Instance of 'IslamicHolidays' has no '_year' member

(E1101)